tests/models/autoencoders/test_models_vae.py

```diff
@@ -810,6 +810,43 @@ def test_stable_diffusion_model_local(self):
 
         assert torch_all_close(output_slice_1, output_slice_2, atol=3e-3)
 
+    def test_single_file_component_configs(self):
+        vae_single_file = AutoencoderKL.from_single_file(
+            "https://huggingface.co/stabilityai/sd-vae-ft-mse-original/blob/main/vae-ft-mse-840000-ema-pruned.safetensors"
+        )
+        vae = AutoencoderKL.from_pretrained("CompVis/stable-diffusion-v1-4", subfolder="vae")
```

Comment:
Not a blocker but here we're making the comparisons with a VAE that has a different set of parameters. That is not the case for the other components we're testing. But since https://huggingface.co/stabilityai/sd-vae-ft-mse-original doesn't have the diffusers-formatted checkpoint available anyway, it's fine here.

Reply:
Yeah 100 percent.